### PR TITLE
Add minor style tweaks for smaller mobile screens

### PR DESCRIPTION
**Pull request by timmybytes**
Hey there! I added a few style tweaks to the CSS to accommodate for smaller mobile screens, including preventing the header logo from being cutoff, and adding Flexbox in a couple places to help keep the layout responsive.

### CODE REVIEW

File: docs/custom.css [hunk 1]
```diff
@@ -3,7 +3,10 @@ html {
 }
 
 body {
-    min-width: 460px;
+    /* ADDED - Add Flexbox for responsiveness */
```
Comment (colinodell) [inline docs/custom.css:6]: Could you please these `/* ADDED */` comments?  Git will keep track of what you added and why :wink:

**Review by colinodell — CHANGES_REQUESTED** [review]
Overall these changes look great!  If you wouldn't mind making that one small change we'll get this merged :)

File: docs/custom.css (hunk 1 above)
Comment (timmybytes) [reply docs/custom.css:6]: Absolutely! I actually originally left these as notes to myself and meant to remove them 😅.

**Review by colinodell — APPROVED** [review]
Thanks so much for the contribution, I really appreciate it!

**Pull request merged**